doc/sphinx/source/data/new-commondata.rst

```diff
@@ -0,0 +1,155 @@
+Organization following the naming convention
+--------------------------------------------
+
+All dataset in the new data format follow the exact same naming convention::
+
+    <experiment>_<process>_<energy>{_<extras>}_<observable>
+
+The data is contained in folders, each containing one single hepdata publication. 
```

Comment:
I assume we can do other things that are not hepdata?

Reply:
Of course vp doesn't know where the data come from but I think the idea is that you open a folder and it comes from one single hepdata publication.

In the odd case in which it doesn't we hope the person implement it add comments or something explaining what happened there.

Reply:
The comment was more in the direction that these docs should be rather general.